### ISSUE TITLE
Error While Setting up

### ISSUE THREAD

**Issue by SahilLangoo** [pankod/superplate#250]
``` 
PS C:\> npx superplate-cli mp
√ Select your project type » Next.js
⠹ Remote source found. Cloning...(node:12504) UnhandledPromiseRejectionWarning: Error: Error: Command failed: git clone https://github.com/pankod/superplate-core-plugins.git C:\Users\Sahil Ahmed\AppData\Local\Temp\d-2021426-12504-1fr3xxp.sefn --depth 1
fatal: Too many arguments.

usage: git clone [<options>] [--] <repo> [<dir>]

    -v, --verbose         be more verbose
    -q, --quiet           be more quiet
    --progress            force progress reporting
    -n, --no-checkout     don't create a checkout
    --bare                create a bare repository
    --mirror              create a mirror repository (implies bare)
    -l, --local           to clone from a local repository
    --no-hardlinks        don't use local hardlinks, always copy
    -s, --shared          setup as shared repository
    --recurse-submodules[=<pathspec>]
                          initialize submodules in the clone
    --recursive ...       alias of --recurse-submodules
    -j, --jobs <n>        number of submodules cloned in parallel
    --template <template-directory>
                          directory from which templates will be used
    --reference <repo>    reference repository
    --reference-if-able <repo>
                          reference repository
    --dissociate          use --reference only while cloning
    -o, --origin <name>   use <name> instead of 'origin' to track upstream
    -b, --branch <branch>
                          checkout <branch> instead of the remote's HEAD
    -u, --upload-pack <path>
                          path to git-upload-pack on the remote
    --depth <depth>       create a shallow clone of that depth
    --shallow-since <time>
                          create a shallow clone since a specific time
    --shallow-exclude <revision>
                          deepen history of shallow clone, excluding rev
    --no-tags             don't clone any tags, and make later fetches not to follow them
    --separate-git-dir <gitdir>
                          separate git dir from working tree
    -c, --config <key=value>
                          set config inside the new repository
    --server-option <server-specific>
                          option to transmit
    -4, --ipv4            use IPv4 addresses only
    -6, --ipv6            use IPv6 addresses only
    --filter <args>       object filtering
    --remote-submodules   any cloned submodules will use their remote-tracking branch
    --sparse              initialize sparse-checkout file to include only files at root


    at Object.CloneAndGetPath (C:\src\node\_npx\68a94a7f4a156bcd\node_modules\superplate-cli\lib\Helper\git\index.js:43:13)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async get_source (C:\src\node\_npx\68a94a7f4a156bcd\node_modules\superplate-cli\lib\Helper\source\index.js:64:29)
    at async cli (C:\src\node\_npx\68a94a7f4a156bcd\node_modules\superplate-cli\lib\cli.js:84:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:12504) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:12504) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
⠏ Remote source found. Cloning...Terminate batch job (Y/N)? y
```
**It throws a long error, takes long time to clone then.
I'm using node v14.16.1 and npm 7.14.0**

**Comment by omeraplak** [comment]
@SahilLangoo Hi,
Thank you for contacting us. We've fixed your issue and release 1.1.6. Could you try again?

**Comment by SahilLangoo** [comment]
Hey
Your welcome, Yes working buttery smooth 🚀  
Thanks for the fix!
